.github/workflows/Unit-Test.yml

```diff
@@ -57,7 +57,7 @@ jobs:
             ruby ../tests/commands/const.rb && \
             ruby ../tests/commands/help.rb && \
             ruby ../tests/commands/prop.rb && \
-            ( [[ -z "${ALL_CHANGED_FILES}" ]] && ( for file in ${ALL_CHANGED_FILES}; do
+            ( [[ -v ALL_CHANGED_FILES ]] && ( for file in ${ALL_CHANGED_FILES}; do
```

Comment:
Don't we want `[[ -n "${ALL_CHANGED_FILES}" ]] && ...`?  We're checking if there are files, correct?

Reply:
The variable doesn't exist if it is empty.

Reply:
```
    String operators:
    
      -z STRING      True if string is empty.
    
      -n STRING
         STRING      True if string is not empty.
```

Reply:
We can try your way but I know that `-n` definitely should work.